go.mod

```diff
@@ -3,34 +3,77 @@ module github.com/openshift/insights-operator
+	github.com/kubernetes-sigs/kube-storage-version-migrator v0.0.0-20191127225502-51849bc15f17 // indirect
+	github.com/morikuni/aec v1.0.0 // indirect
+	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
+	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
+	github.com/opencontainers/image-spec v1.0.1 // indirect
+	github.com/opencontainers/runc v0.0.0-20191031171055-b133feaeeb2e // indirect
 	github.com/openshift/api v0.0.0-20200326160804-ecb9283fe820
 	github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0
-	github.com/openshift/library-go v0.0.0-20200421122923-c1de486c7d47
-	github.com/spf13/cobra v0.0.5
+	github.com/openshift/library-go v0.0.0-00010101000000-000000000000
+	github.com/pkg/profile v1.3.0 // indirect
+	github.com/pquerna/cachecontrol v0.0.0-20171018203845-0dec1b30a021 // indirect
+	github.com/prometheus/client_model v0.2.0 // indirect
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 // indirect
-	golang.org/x/net v0.0.0-20200301022130-244492dfa37a
-	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
-	google.golang.org/appengine v1.6.1 // indirect
-	k8s.io/api v0.18.9
-	k8s.io/apiextensions-apiserver v0.18.9
-	k8s.io/apimachinery v0.18.9
-	k8s.io/client-go v11.0.0+incompatible
-	k8s.io/component-base v0.18.9
+	github.com/vishvananda/netlink v1.0.0 // indirect
+	github.com/vishvananda/netns v0.0.0-20191106174202-0a2b9b5464df // indirect
+	github.com/xlab/handysort v0.0.0-20150421192137-fb3537ed64a1 // indirect
+	golang.org/x/net v0.0.0-20201002202402-0a1ea396d57c
+	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e // indirect
+	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
+	gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d // indirect
+	gopkg.in/ldap.v2 v2.5.1 // indirect
+	gopkg.in/natefinch/lumberjack.v2 v2.0.0 // indirect
+	gopkg.in/square/go-jose.v2 v2.2.2 // indirect
+	gotest.tools v2.2.0+incompatible // indirect
+	k8s.io/api v0.19.4
+	k8s.io/apiextensions-apiserver v0.19.4
+	k8s.io/apimachinery v0.19.4
+	k8s.io/client-go v0.19.4
+	k8s.io/component-base v0.19.4
 	k8s.io/klog v1.0.0
+	vbom.ml/util v0.0.0-20180919145318-efcd4e0f9787 // indirect
 )
 
 replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20200326160804-ecb9283fe820
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0
 	github.com/openshift/library-go => github.com/openshift/library-go v0.0.0-20200421122923-c1de486c7d47
-	k8s.io/api => k8s.io/api v0.18.9
-	k8s.io/apimachinery => k8s.io/apimachinery v0.18.9
-	k8s.io/apiserver => k8s.io/apiserver v0.18.9
-	k8s.io/client-go => k8s.io/client-go v0.18.9
+	k8s.io/api => k8s.io/api v0.19.4
+	k8s.io/apimachinery => k8s.io/apimachinery v0.19.4
+	k8s.io/apiserver => k8s.io/apiserver v0.19.4
+	k8s.io/client-go => k8s.io/client-go v0.19.4
```

Comment:
I don't think we want to update the K8s API in this PR.

Reply:
Without that, there's no way to test it(streaming logs is not implemented in the previous version), so we need to either remove test and revert dependencies or keep it as is.

Reply:
Well OK. Openshift API is using K8s API `v0.19.2` (see https://github.com/openshift/api/blob/master/go.mod) so we can probably update to the same version, but I would split it into 2 PRs - one doing the versions update and second the actual log gathering. 

@natiiix Any recommendations for the version updates? I know you've already done it.

Reply:
I just found out that the API is about to be updated to 1.20 (not sure when and 1.20 looks like to be equal to v0.20) so if we don't want to wait we will need to do the update twice....but maybe we can just stay on the 0.19.2

Reply:
I mean, streaming logs is in "k8s.io/client-go", but we need to update other dependencies as well since there will be a dependency conflict.